### PR TITLE
Switch from local readiness check to script found in SCF repo.

### PR DESCRIPTION
**Pull request by andreas-kupries**
- Removed local script
- Added resource for scf repo
- Updated task spec and script to use the script from the scf repo/resource.
- Added helper for deploying the pipeline

### CODE REVIEW

File: setup-cf-deploy-test-teardown.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#           pipelines
+
+if [ ! -f "${secrets}" ]
+then
+    echo -e 1>&2 "$0: Failed to find the secrets file ${secrets}\n\tPlease specify the correct directory holding \"${secfile}\"."
+    exit 1
+fi
+
+fly -t "$target" set-pipeline \
+    -p ${PP}cf-deploy-test-teardown \
```
Comment (alexjh) [inline setup-cf-deploy-test-teardown.sh]: I really like this option for adding a prefix but I think it should have a more descriptive name.

File: cf-deploy-test-teardown/cf-deploy-test-teardown.yml [hunk 2]
```diff
@@ -1,6 +1,14 @@
 # Concourse pipeline to build deploy and teardown cf against a kube cluster
 
 resources:
+- name: src
+  type: git
+  source:
+    uri: https://github.com/SUSE/scf.git
+    branch: develop
+    paths:
+    - bin/*
```
Comment (alexjh) [inline cf-deploy-test-teardown/cf-deploy-test-teardown.yml]: The only file it needs to check is `bin/dev/kube-ready-state-check.sh`, I don't see any reason to use a wildcard here.

Comment (andreas-kupries) [inline cf-deploy-test-teardown/cf-deploy-test-teardown.yml]: Will fix.

File: setup-cf-deploy-test-teardown.sh (hunk 1 above)
Comment (andreas-kupries) [inline setup-cf-deploy-test-teardown.sh]: I know I am bad with names. I currently have nothing better. Do you ?

File: cf-deploy-test-teardown/cf-deploy-test-teardown.yml (hunk 2 above)
Comment (andreas-kupries) [inline cf-deploy-test-teardown/cf-deploy-test-teardown.yml]: Done with commit 20b2ceb1881ee5d4882e368998cbf5e2059a40ca

File: setup-cf-deploy-test-teardown.sh (hunk 1 above)
Comment (alexjh) [inline setup-cf-deploy-test-teardown.sh]: It is one of the two hard problems in computing. :)

How about PIPELINE_PREFIX? I know it's long to type, but I think I'd typically export it once for a testing session, so it's not like you'd be typing it often.

Comment (andreas-kupries) [inline setup-cf-deploy-test-teardown.sh]: True. :+1: on the name, fix coming up ...

Comment (andreas-kupries) [inline setup-cf-deploy-test-teardown.sh]: Done with commit 12fe81f009d764caa3b3846b3bdc62efceee049c

**Pull request merged**